internal/drivers/postgresql/postgresql.go

```diff
@@ -218,6 +227,17 @@ func (p *postgresqlDriver) Import(config internal.ImporterConfig) error {
 		if size > 0 {
 			if err := executeSQL(pending.String()); err != nil {
 				logger.Trace("offending sql: %s", pending.String())
+				//Post the offending sql to a file log. For large batch inserts, the offending SQL takes up too much space in the terminal
+				file, fileErr := os.Create("offend.txt")
```

Comment:
might want to just create a temp file and then print out the print file path as part of the error message...